Mirror/Runtime/NetworkBehaviour.cs

```diff
@@ -48,6 +48,22 @@ NetworkIdentity myView
             }
         }
 
+        public byte ComponentIndex { 
+            get 
+            {
+                NetworkBehaviour[] components = GetComponents<NetworkBehaviour>();
```

Comment:
what about myView.m_NetworkBehaviours here? would avoid finding them again

Reply:
it is private

Reply:
also,  there is no guarantee that m_NetworkBehaviors is populated,   it is just a cache.